characterisation/exceptions/HO100108.test.ts

```diff
@@ -1,41 +1,45 @@
-jest.setTimeout(30000)
+jest.setTimeout(30000);
 
-import World from "../../steps/world"
-import type { GenerateMessageOptions } from "../helpers/generateMessage"
-import generateMessage from "../helpers/generateMessage"
-import processMessage from "../helpers/processMessage"
+import World from "../../steps/world";
```

Comment:
Has the eslint config changed? Why do we have all these semicolons?

Reply:
I see, the eslint config is different in the core repo, maybe we can fix this in a separate PR